sections/futures/Trade/FuturesUnsupported.tsx

```diff
@@ -0,0 +1,63 @@
+import { useConnectModal } from '@rainbow-me/rainbowkit';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import useNetworkSwitcher from 'hooks/useNetworkSwitcher';
+import { BorderedPanel } from 'styles/common';
+
+type FuturesUnsupportedProps = {
+	isWalletConnected: boolean;
+};
+
+const FuturesUnsupported: React.FC<FuturesUnsupportedProps> = ({ isWalletConnected }) => {
+	const { t } = useTranslation();
+	const { switchToL2 } = useNetworkSwitcher();
+	const { openConnectModal: connectWallet } = useConnectModal();
+	return (
+		<MessageContainer>
+			<Title>{t('futures.page-title')}</Title>
+			{isWalletConnected ? (
+				<>
+					<UnsupportedMessage>{t('common.l2-cta')}</UnsupportedMessage>
+					<IsolatedLink>
+						<div onClick={switchToL2}>{t('homepage.l2.cta-buttons.switch-l2')}</div>
+					</IsolatedLink>
+				</>
+			) : (
+				<>
+					<UnsupportedMessage>{t('common.perp-cta')}</UnsupportedMessage>
+					<IsolatedLink>
+						<div onClick={connectWallet}>{t('common.wallet.connect-wallet')}</div>
+					</IsolatedLink>
+				</>
+			)}
+		</MessageContainer>
+	);
+};
+
+const UnsupportedMessage = styled.div`
+	margin-top: 12px;
+`;
+
+const IsolatedLink = styled.div`
```

Comment:
I think we can rename this to just Link or something more generic.

Reply:
We use `Link` from `next/link` in the code repo. So maybe name this as `LinkContainer`? 
![image](https://user-images.githubusercontent.com/4819006/191973244-25bc62f2-bbf4-4202-83b6-97171899b53a.png)

Reply:
Yea sounds good

Reply:
Done.